mergify_engine/actions/__init__.py

```diff
@@ -85,6 +85,15 @@ class Action(abc.ABC):
     # not create something that endup with a infinite loop of events
     allow_retrigger_mergify = False
 
+    BACKPORT = "backport"
+    COPY = "copy"
+
+    BRANCH_PREFIX_MAP = {BACKPORT: "bp", COPY: "copy"}
+
+    @staticmethod
+    def branch_prefix(kind):
+        return Action.BRANCH_PREFIX_MAP[kind]
```

Comment:
Maybe just add a `BRANCH_PREFIX` in each action class

Reply:
The rest looks good, I will move this myself, squash all commits, and generates the final fixtures.

Thanks, @v1v!